packages/web-components/fast-foundation/docs/guide/aurelia.doc.md

```diff
@@ -3,4 +3,200 @@ id: aurelia
 title: Aurelia
 sidebar_label: Aurelia
 custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/packages/web-components/fast-foundation/docs/guide/aurelia.doc.md
----
\ No newline at end of file
+---
+
+FAST-DNA works flawlessly with both Aurelia 1 and Aurelia 2, with full integration into the binding engine and component model. Let's take a look at how you can set up an Aurelia project, starting from scratch.
+
+## Aurelia 2
+
+### Setting up the Aurelia 2 Project
+
+First, you'll need to make sure that you have Node.js installed. You can learn more and download that [on the official site](https://nodejs.org/).
+
+With Node.js installed, you can run the following command to create a new Aurelia 2 project:
+
+```shell
+npx makes Aurelia
+```
+
+Follow the prompts, answering each question in turn. It is recommended that you select the "Default TypeScript Aurelia 2 App" when prompted, unless you have previous experience with the CLI. Be sure to choose to install dependencies when asked.
+
+When the CLI completes, you should have a basic runnable Aurelia 2 application.
+
+### Configuring Packages
+
+Next, we'll install the FAST-DNA packages, along with supporting libraries. To do that, run this command from your new project folder:
+
+```shell
+npm install --save @microsoft/fast-components @microsoft/fast-element lodash-es
+```
+
+## Using the Components
+
+With all the basic pieces in place, let's run our app in dev mode with `npm start`. Webpack should build your project and open your default browser with your `index.html` page. Right now, it should only have a hello message, since we haven't added any code or interesting HTML. Let's change that.
+
+First, open your `src/main.ts` file and add the following code:
+
+```ts
+import { 
+  FASTDesignSystemProvider, 
+  FASTCard, 
+  FASTButton 
+} from '@microsoft/fast-components';
+
+FASTDesignSystemProvider;
+FASTCard;
+FASTButton;
+```
+
+This code imports the `<fast-design-system-provider>` component as well as the `<fast-card>`, and `<fast-button>` components. Once you save, the dev server will rebuild and refresh your browser. However, you still won't see anything. To get some UI showing up, we need to write some HTML that uses our components. Replace your `my-app.html` file with the following markup:
+
+```html
+<fast-design-system-provider use-defaults>
+  <fast-card>
+    <h2>${message}</h2>
+    <fast-button appearance="accent"
+                 click.trigger="onClick()">Click Me</fast-button>
```

Comment:
The indention on row 59 looks out of place.  Is this intentional?  Maybe don't even wrap it?

Reply:
I did it on purpose but there was also additional attributes when I made the formatting. I can remove the indent.